src/app/util/ember-compatibility-functions.cpp

```diff
@@ -516,6 +516,27 @@ Protocols::InteractionModel::Status UnsupportedAttributeStatus(const ConcreteAtt
 
 } // anonymous namespace
 
+bool CheckConcreteAttributePathExistence(const AttributePathParams & aPath)
+{
+    const EmberAfAttributeMetadata * attributeMetadata = nullptr;
+    const EmberAfCluster * attributeCluster            = nullptr;
+
+    switch (aPath.mAttributeId)
+    {
+    case Clusters::Globals::Attributes::AttributeList::Id:
```

Comment:
Please use `GlobalAttributesNotInMetadata` from `src/app/GlobalAttributes.h` instead of hardcoding a second copy of that list.

We should fix `ReadSingleClusterData` to do that too, in a follow-up.

Reply:
updated, would follow-up in ReadSingleClusterData
https://github.com/project-chip/connectedhomeip/pull/22349